Jenkinsfile

```diff
@@ -70,103 +55,189 @@ pipeline {
         }
       }
     }
-    stage('Dockerize SoQL Server PG') {
+    stage('Docker Build') {
       when {
         not { expression { isPr } }
       }
-      steps {
-        script {
-          lastStage = env.STAGE_NAME
-          if (params.RELEASE_BUILD) {
-            env.REGISTRY_PUSH = (params.RELEASE_DRY_RUN) ? 'none' : 'all'
-            env.DOCKER_TAG = dockerize_server.docker_build_specify_tag_and_push(params.RELEASE_NAME, sbtbuild.getDockerPath(project_wd_server), sbtbuild.getDockerArtifact(project_wd_server), env.REGISTRY_PUSH)
-          } else {
-            env.REGISTRY_PUSH = 'internal'
-            env.DOCKER_TAG = dockerize_server.docker_build('STAGING', env.GIT_COMMIT, sbtbuild.getDockerPath(project_wd_server), sbtbuild.getDockerArtifact(project_wd_server), env.REGISTRY_PUSH)
+      parallel {
+        stage ('Docker Build - SoQL Server') {
+          steps {
+            script {
+              lastStage = env.STAGE_NAME
+              if (params.RELEASE_BUILD) {
+                env.DOCKER_TAG = dockerize_server.dockerBuildWithSpecificTag(
```

Comment:
New methods dockerBuildWithSpecificTag and dockerBuildWithDefaultTag (see https://github.com/socrata/pipelines/blob/main/src/com/socrata/Dockerize.groovy)
For each one, the parameter is a map, which can be entered without the outside brackets.
They return just the docker tag (`H24` or `staging_183_289dkkck`).

Reply:
See https://github.com/socrata/activity-feed/pull/188 for example usage.